terraform/vsphere/README.md

```diff
@@ -0,0 +1,44 @@
+# automate-backend-vsphere-module
+
+Terraform Module
+
+## Module Dependencies
+None
+
+## Description
+Creates a specified number of servers labelled as Postgres, Elasticsearch and Automate on your vSphere cluster.
+- It does no configuration of the machines, except for mounting and formatting the data disk
+- It expects the vSphere cluster to have HA and DRS enabled, rather than selecting compute and storage resources for each VM.
+
+## Example usage
```

Comment:
We may need more of a "just run this command" like `terraform apply` level how to description or at least need this to be easily understood on how to get started. I'm not sure how to use modules or what I need to do here.